[bug] OSV Certifier unit tests failing due to mismatched output

CI is failing . The test should be updated to match upstream changes.

```
=== FAIL: pkg/certifier/osv TestOSVCertifier_CertifyVulns/ensure_intermediate_vulnerabilities_are_reported (0.16s)
    osv_test.go:218: g.RetrieveArtifacts() = {"type":"https://in-toto.io/Statement/v1","subject":[{"uri":"pkg:maven/io.vertx/vertx-core@4.3.7?type=jar"}],"predicate_type":"https://in-toto.io/attestation/vulns/v0.1","predicate":{"scanner":{"uri":"osv.dev","version":"0.0.14","db":{},"result":[{"id":"GHSA-9ph3-v2vh-3qx7"},{"id":"GHSA-cphf-4846-3xx9"}]},"metadata":{"scanStartedOn":"2026-02-09T15:58:30.797127394Z","scanFinishedOn":"2026-02-09T15:58:30.797127394Z"}}}, want {
        		"type": "https://in-toto.io/Statement/v1",
        		"subject": [
        			{
        				"uri": "pkg:maven/io.vertx/vertx-core@4.3.7?type=jar"
        			}
        		],
        		"predicate_type": "https://in-toto.io/attestation/vulns/v0.1",
        		"predicate": {
        			"scanner": {
        				"uri": "osv.dev",
        				"version": "0.0.14",
        				"result": [
        					{
        						"id": "GHSA-9ph3-v2vh-3qx7"
        					}
        				]
        			},
        			"metadata": {
        				"scanStartedOn":"2023-02-15T11:10:08.986506-08:00",
        				"scanFinishedOn":"2023-02-15T11:10:08.986506-08:00"
        			}
        		}
        	}
    --- FAIL: TestOSVCertifier_CertifyVulns/ensure_intermediate_vulnerabilities_are_reported (0.16s)

```

--- a/internal/testing/testdata/testdata.go
+++ b/internal/testing/testdata/testdata.go
@@ -2392,12 +2392,15 @@ var (
 				"result": [
 					{
 						"id": "GHSA-9ph3-v2vh-3qx7"
+					},
+					{
+						"id":"GHSA-cphf-4846-3xx9"
 					}
 				]
 			},
 			"metadata": {
-				"scanStartedOn":"2023-02-15T11:10:08.986506-08:00",
-				"scanFinishedOn":"2023-02-15T11:10:08.986506-08:00"
+				"scanStartedOn":"2026-02-09T15:58:30.797127394Z",
+				"scanFinishedOn":"2026-02-09T15:58:30.797127394Z"
 			}
 		}
 	}`

